Dockerfile

```diff
@@ -66,7 +66,7 @@ RUN BUILD_DEPS="build-essential \
 
 ## Djatoka
 RUN cd /tmp && \
-    curl -O https://sourceforge.mirrorservice.org/d/dj/djatoka/djatoka/1.1/adore-djatoka-1.1.tar.gz && \
+    curl -LO http://downloads.sourceforge.net/project/djatoka/djatoka/1.1/adore-djatoka-1.1.tar.gz && \
```

Comment:
Hi Gavin, 
I've noticed you've changed a few of these URL's - is it just preference, or is this a faster download location? 

Thanks,
Bethany